eq-author/src/redux/auth/actions.js

```diff
@@ -4,14 +4,23 @@ export const SIGN_IN_USER = "SIGN_IN_USER";
 export const SIGN_OUT_USER = "SIGN_OUT_USER";
 export const SIGN_IN_ERROR = "SIGN_IN_ERROR";
 
-export const signInUser = ({ displayName, email, photoURL }) => {
+export const signInUser = ({
+  displayName,
+  email,
+  photoURL,
+  stsTokenManager: { accessToken },
```

Comment:
What is `stsTokenManager` used for?

Reply:
Oh this like makes the json onto parameters

Reply:
`getIdToken` may tie us in with Firebase so might be worth handling this at a later date in a more generic way.